### PR TITLE
updated workflow to test upload to s3 bucket

### PR DESCRIPTION
**Pull request by jspaaks**
In this PR I added uploading to S3 as a workflow step. While we're testing, I also limited the number of urls for which to run the analysis

### CODE REVIEW

File: .github/workflows/tortellini.yml [hunk 1]
```diff
@@ -8,10 +8,27 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-    - run: curl https://research-software.nl/api/software_cache | jq '.[] | .repositoryURLs.github[]' --raw-output > urls.txt
+    - run: |
+        curl https://research-software.nl/api/software_cache | jq '.[] | .repositoryURLs.github[]' --raw-output > urls-all.txt
+        cat urls-all.txt | tail -n 2 > urls.txt
+        
+    - run: mkdir .tortellini
```
Comment (fdiblen) [inline .github/workflows/tortellini.yml]:
```suggestion
```

File: .github/workflows/tortellini.yml [hunk 2]
```diff
@@ -8,10 +8,27 @@ jobs:
...
 
-    - uses: tortellini-tools/action@main
+    - uses: tortellini-tools/action@36-vue-app
```
Comment (fdiblen) [inline .github/workflows/tortellini.yml]:
```suggestion
    - uses: tortellini-tools/action@main
```

File: .github/workflows/tortellini.yml [hunk 3]
```diff
@@ -8,10 +8,27 @@ jobs:
...
       with:
         repositories: urls.txt
         
     - run: tree -L 3
+    
+    - run: sudo rm -rf .tortellini/in/*
```
Comment (fdiblen) [inline .github/workflows/tortellini.yml]:
```suggestion
```

File: .github/workflows/tortellini.yml [hunk 4]
```diff
@@ -8,10 +8,27 @@ jobs:
...
+    - uses: jakejarvis/s3-sync-action@master
+      with:
+        args: --acl public-read --follow-symlinks --delete
+      env:
+        AWS_S3_BUCKET: ${{ secrets.AWS_S3_BUCKET }}
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        AWS_REGION: 'eu-central-1'   # optional: defaults to us-east-1
+        SOURCE_DIR: '.tortellini'      # optional: defaults to entire repository    
+        DEST_DIR: '.tortellini'
```
Comment (fdiblen) [inline .github/workflows/tortellini.yml]:
```suggestion
```

File: .github/workflows/tortellini.yml [hunk 5]
```diff
@@ -8,10 +8,27 @@ jobs:
...
+        AWS_S3_BUCKET: ${{ secrets.AWS_S3_BUCKET }}
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        AWS_REGION: 'eu-central-1'
+        SOURCE_DIR: '.tortellini'
+        
+    - name: See the results
+      run: |
+        echo -n 'go to "https://tortellini.research'
+        echo -n '-software.nl/index.html" to see the results.'
```
Comment (fdiblen) [inline .github/workflows/tortellini.yml:34]:
```suggestion
```

File: .github/workflows/tortellini.yml [hunk 6]
```diff
@@ -8,10 +8,27 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-    - run: curl https://research-software.nl/api/software_cache | jq '.[] | .repositoryURLs.github[]' --raw-output > urls.txt
+    - run: |
+        curl https://research-software.nl/api/software_cache | jq '.[] | .repositoryURLs.github[]' --raw-output > urls-all.txt
+        cat urls-all.txt | tail -n 2 > urls.txt
```
Comment (fdiblen) [inline .github/workflows/tortellini.yml]:
```suggestion
        curl https://research-software.nl/api/software_cache | jq '.[] | .repositoryURLs.github[]' --raw-output > urls.txt
```

File: .github/workflows/tortellini.yml (hunk 5 above)
Comment (jspaaks) [inline .github/workflows/tortellini.yml:33]: I'd like to keep this

**Review by fdiblen — APPROVED** [review]
I saw it working with my own eyes. A report has a very nice overview. Thank you @jspaaks 🧾

**Comment by jspaaks** [comment]
Woohoo

**Pull request merged**